### DISCUSSION TITLE
i18n finnish "Suomi"

### DISCUSSION THREAD

**Discussion by kshji** [McShelby/hugo-theme-relearn#355] [Ideas]
I done finnish (fi) translating to my own fork. Take translate to finnish.
[fi.toml finnish - fi](https://github.com/kshji/hugo-theme-relearn/blob/main/i18n/fi.toml).

**Comment by McShelby** [comment]
Could you please make a pull request from your forked repository. By that, I am able to simply merge these changes into the release.
Thanks for contributing.
